kernel/src/process/process/mod.rs

```diff
@@ -815,3 +825,26 @@ pub fn broadcast_signal_async(process_group: Weak<ProcessGroup>, signum: SigNum)
         work_queue::WorkPriority::High,
     );
 }
+
+/// The resource usage statistics of child processes that have terminated and
+/// been reaped by the parent.
+///
+/// These statistics include the resources consumed by grandchildren and more
+/// distant descendants, if all intermediate child processes have waited on
+/// their own terminated children.
+#[derive(Default)]
+pub struct ReapedChildrenStats {
+    cutime: Duration,
+    cstime: Duration,
+}
```

Comment:
Since this is used inside the kernel module, we don't need to use`cutime` and `cstime` to follow the Linux names, which does not clearly convey their meanings. I suggest renaming them as `kernel_time` and `user_time` for clearer meaning. Other places may need also to be refactoered.